src/modules/proposal/entities/proposal.entity.ts

```diff
@@ -37,4 +39,13 @@ export class Proposal {
     enum: ProposalType,
   })
   type: ProposalType;
+
+  @JoinColumn({ name: 'fileId' })
+  @OneToOne(() => LocalFile, {
+    nullable: true,
+  })
+  file?: LocalFile;
+
+  @Column({ nullable: true })
+  fileId?: number;
```

Comment:
why we need file and fileId?

Reply:
It was redundant, proposal entity and create service fixed